CourseTemplate/CourseTemplate.Core/CourseTemplate.Core.csproj

```diff
@@ -7,7 +7,14 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\..\AcademicHub.Common\CustomExceptions\CustomExceptions.csproj" />
       <ProjectReference Include="..\CourseTemplate.Data\CourseTemplate.Data.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
This can be removed. If you reference csproj, you don't need dll reference.
It will also solve all problems with build: you'll literally don't need any CI action for it.

Reply:
![Uploading Screenshot 2023-11-17 at 17.01.50.png…]()